source/includes/usage-examples/code-snippets/ReplaceOne.java

```diff
@@ -0,0 +1,47 @@
+// ignored first line
+package usage.examples;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoCursor;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.ReplaceOptions;
+import com.mongodb.client.result.UpdateResult;
+
+public class ReplaceOne {
+
+    public static void main(String[] args) {
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+        MongoClient mongoClient = MongoClients.create(uri);
+
+        MongoDatabase database = mongoClient.getDatabase("sample_mflix");
+        MongoCollection<Document> collection = database.getCollection("movies");
+
+        try {
```

Comment:
Can't remember if we discussed this already, but should we use try-with-resources here e.g.

    try(MongoClient mongoClient = MongoClients.create(uri))
         MongoDatabase database = mongoClient.getDatabase("sample_mflix");
         MongoCollection<Document> collection = database.getCollection("movies");
         ...

Reply:
It's more modern Java

Reply:
Thanks, will update. I now recall we discussed try-with-resources in the context of opening a file for the CSFLE guide. Will fix for this PR and revisit other code examples in docs-java.